src/dispatch/case/type/service.py

```diff
@@ -148,6 +160,13 @@ def create(*, db_session, case_type_in: CaseTypeCreate) -> CaseType:
 
 def update(*, db_session, case_type: CaseType, case_type_in: CaseTypeUpdate) -> CaseType:
     """Updates a case type."""
+    cost_model = None
+    if case_type_in.cost_model:
+        cost_model = cost_model_service.get_cost_model_by_id(
+            db_session=db_session, cost_model_id=case_type_in.cost_model.id
+        )
+    case_type.cost_model = cost_model
+
```

Comment:
incident/type/service.py also includes a check for `should_update_incident_cost` - should we do that here too?

Reply:
Oh. it looks like i accidentally added the service.py file to this PR.
It's meant to be in https://github.com/Netflix/dispatch/pull/4900